tests/kernel/profiling/profiling_api/src/main.c

```diff
@@ -88,9 +97,10 @@ static void thread_callback(const struct k_thread *thread, void *user_data)
 		thread_flag = true;
 	}
 
+
 	if (((char *)thread->stack_info.start ==
-			K_THREAD_STACK_BUFFER(test_stack)) &&
-			(thread->stack_info.size == STACKSIZE)) {
+			K_THREAD_STACK_BUFFER(test_stack))
+			&& thread->stack_info.size) {
```

Comment:
You could probably just drop the check of stack_info.size all together.

Reply:
Ok will do.